docs/content/en/docs/tasks/cluster/cluster-upgrades/vsphere-and-cloudstack-upgrades.md

```diff
@@ -13,6 +13,21 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 
 **Upgrades should never be run from ephemeral nodes (short-lived systems that spin up and down on a regular basis). If an upgrade fails, it is very important not to delete the Docker containers running the KinD bootstrap cluster. During an upgrade, the bootstrap cluster contains critical EKS Anywhere components. If it is deleted after a failed upgrade, they cannot be recovered.**
 
+{{% alert title="Important" color="warning" %}}
+
+In `eksctl anywhere` version `v0.13.0`, we introduced the full lifecycle controller to fully manage new workload clusters.
+The controller behavior would roll out new nodes in the workload cluster whenever the user upgraded the management cluster and its management components.
+
+In `eksctl anywhere` version `v0.16.0`, we changed this behavior to allow users to be explicit when deciding which clusters to upgrade.
+Therefore, workload clusters are no longer affected by management cluster upgrades.
+Due to this change, each cluster must be individually upgraded to enjoy the latest features on all clusters.
+If you have a management cluster running EKS Anywhere version v0.15, you can successfully upgrade to EKS Anywhere version v0.16 and observe no changes to any of its workload clusters.
+
+When triggering a workload cluster upgrade after upgrading the management cluster, please keep in mind that it will not only reflect your changes in the cluster spec, but also any new improvements included in the new EKS-A controller.
```

Comment:
```suggestion
When triggering a workload cluster upgrade after upgrading the management cluster, please keep in mind that it will not only reflect your changes in the cluster spec, but also any new improvements included in the new EKS Anywhere controller.
```